components/cluster/command/display.go

```diff
@@ -91,7 +92,14 @@ func displayDashboardInfo(clusterName string) error {
 		pdEndpoints = append(pdEndpoints, fmt.Sprintf("%s:%d", pd.Host, pd.ClientPort))
 	}
 
-	pdAPI := api.NewPDClient(pdEndpoints, 2*time.Second, nil)
+	var tlsCfg *tls.Config
```

Comment:
There is so much duplicated code to check whether the `metadata.Topology.GlobalOptions.TLSEnabled` is true. It's better to define a method for `Topology`, e.g:
```
func (t *Topology) TSLConfig(path string)  (*tsl.Config, error) {
    if !t.GlobalOptions.TLSEnabled {
        return nil, nil
    }
    tlsCfg, err = spec.LoadClientCert(path)
    if err != nil {
        return perrs.AddStack(err)
    }
}

tslConfig, err := metadata.Topology.TSLConfig(tidbSpec.Path(clusterName, spec.TLSCertKeyDir))
```